src/scripts/templates/customers/addresses.js

```diff
@@ -7,56 +7,65 @@
  * @namespace customerAddresses
  */
 
-import $ from 'jquery';
-
-const $newAddressForm = $('#AddressNewForm');
-
-if ($newAddressForm.length) {
-  // Initialize observers on address selectors, defined in shopify_common.js
-  if (Shopify) {
-    new Shopify.CountryProvinceSelector(
-      'AddressCountryNew',
-      'AddressProvinceNew',
-      {
-        hideElement: 'AddressProvinceContainerNew',
-      },
-    );
-  }
+import {CountryProvinceSelector} from '@shopify/theme-addresses';
+
```

Comment:
One convention we typically follow is to place all selectors in a `selectors` object near the top so they can easily be referenced multiple times in the code below:
```
const selectors = {
  addressCountryNew: '#AddressCountryNew',
  addressCountryId: (formId) => `#AddressCountry_${formId}`,
  ...
}